format.go

```diff
@@ -19,6 +19,10 @@ type Resolver interface {
 	Tree(path string, depth int) []string
 }
 
+// Node is the base interface all IPLD nodes must implement.
+//
+// Nodes are **Immutable** and all methods defined on the interface are
+// **Thread Safe**.
```

Comment:
If Nodes are immutable, what is the reason for `Copy` method?

Reply:
None. I was planning on fixing that in this PR but have punted on fixing Nodes till a future one.